e2e/rdctl.e2e.spec.ts

```diff
@@ -21,6 +21,7 @@ limitations under the License.
 import fs from 'fs';
 import path from 'path';
 
+import os from 'os';
 import { expect, test } from '@playwright/test';
```

Comment:
```suggestion
import os from 'os';
import path from 'path';

import { expect, test } from '@playwright/test';
```
(Trivial whitespace fix)

Reply:
Weird that goland didn't suggest this...